pandas/tests/extension/base/dtype.py

```diff
@@ -70,13 +72,13 @@ def test_check_dtype(self, data):
 
         # np.dtype('int64') == 'Int64' == 'int64'
         # so can't distinguish
-        if dtype.name == "Int64":
+        if dtype.name == "Int64" and _np_version_under1p20:
+            # TODO(numpy-1.20): This if block and the warnings filter can be removed
+            # once we require numpy>=1.20
             expected = pd.Series([True, True, False, True], index=list("ABCD"))
         else:
             expected = pd.Series([True, True, False, False], index=list("ABCD"))
 
-        # FIXME: This should probably be *fixed* not ignored.
```

Comment:
I'm not sure that there's anything we can actually do here. Open to suggestions, but the fact is that NumPy controls the behavior of `np.dtype.__eq__`. So I'm comfortable just leaving this for NumPy<1.20.